mlflow/tracking/fluent.py

```diff
@@ -386,6 +388,62 @@ def active_run() -> Optional[ActiveRun]:
     return _active_run_stack[-1] if len(_active_run_stack) > 0 else None
 
 
+def last_active_run() -> Optional[Run]:
+    """
```

Comment:
Can we also update our examples that use autologging to use this API, as well as add some docs about this here?: https://mlflow.org/docs/latest/tracking.html#automatic-logging